namespaces/aushape.yml

```diff
@@ -39,8 +39,8 @@ namespace:
     - name: avc
       type: nested
     - name: execve
-      type: string
-      index: analyzed
+      type: text
+      index: true
```

Comment:
What does `index: true` mean here?

Reply:
@spbnick https://www.elastic.co/guide/en/elasticsearch/reference/5.6/text.html

Reply:
Ah, I see, that's fine. Thank you, @richm!